src/tests/ftest/launch.py

```diff
@@ -1364,7 +1364,8 @@ def install_debuginfos():
     """Install debuginfo packages."""
     install_pkgs = [{'name': 'gdb'}]
 
-    cmds = []
+    cmd_list = []
+    timeout = ["timeout", "-k", "-v", "120"]
```

Comment:
`-v` isn't a valid option for `timeout` on EL7, but even if it were, the `120` needs to come after the `-k` as it's an argument to `-k` not `timeout`.

But the changes below should eliminate the need for it in any case.

Reply:
This has been removed.